src/dune_rules/context.ml

```diff
@@ -376,7 +375,7 @@ let create ~(kind : Kind.t) ~path ~env ~env_nodes ~name ~merlin ~targets
         | Some s -> Memo.return s
         | None ->
           Memo.of_reproducible_fiber
-            (Process.run_capture_line ~display:!Clflags.display ~env Strict fn
+            (Process.run_capture_line ~display:Quiet ~env Strict fn
```

Comment:
Here is another call to ocamlfind to query the context. This is internal and does not need to be reported to the user.